python/ml4ir/applications/ranking/tests/test_metrics.py

```diff
@@ -12,26 +12,26 @@
                 'new_ACR': 2.410,
                 'old_MRR': 0.783,
                 'new_MRR': 0.597,
-                'mean_old_name_match_failure_all': 0.061,
-                'mean_old_name_match_failure_any': 0.111,
-                'mean_old_name_match_failure_all_rank': 0.142,
-                'mean_old_name_match_failure_any_rank': 0.351,
-                'mean_old_name_match_failure_any_count': 0.165,
-                'mean_old_name_match_failure_any_fraction': 0.087,
-                'mean_new_name_match_failure_all': 0.086,
-                'mean_new_name_match_failure_any': 0.218,
-                'mean_new_name_match_failure_all_rank': 0.196,
-                'mean_new_name_match_failure_any_rank': 0.788,
-                'mean_new_name_match_failure_any_count': 0.342,
-                'mean_new_name_match_failure_any_fraction': 0.153,
+                'old_name_match_failure_all_mean': 0.061,
```

Comment:
If you notice, none of the values have changed. I just changed "mean" to the suffix so that it is easier to handle in a generic way.